@DQ/Q8.m

```diff
@@ -1,6 +1,11 @@
-% Return the partial derivative of the unit dual quaternion x with respect to log(x)
-
-% (C) Copyright 2011-2019 DQ Robotics Developers
+% Q8(x) returns the partial derivative of the unit dual quaternion x with
```

Comment:
Given the unit dual quaternion x, Q8(x) returns the partial derivative of vec8(x) with respect to vec6(log(x)).